projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/DataJobViewLogsIT.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
I am probably missing something, but I don't see cleanup of the test job being performed. Does that mean that the job just runs for 20 minutes, and is then left in the namespace?

Reply:
DataJobDeploymentExtension clean it up when the test finished.